projects/kit/package.json

```diff
@@ -14,6 +14,8 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@ng-web-apis/intersection-observer": "3.0.0",
+        "@ng-web-apis/mutation-observer": "2.0.0",
```

Comment:
It is already inside `peerDependencies`

Reply:
We shouldn't add them as dependencies. They should be in peer dependencies. The issue is `@ng-web-apis/resize-observer` is missing there.

Reply:
u r right, thanks